rancher/v1.2/en/cattle/adding-load-balancers/index.md

```diff
@@ -8,70 +8,54 @@ redirect_from:
   - /rancher/latest/en/cattle/adding-load-balancers/
 ---
 
-## Adding Load Balancers
+## Load Balancers
 ---
 
-Rancher implements a managed load balancer using HAProxy that can be manually scaled to multiple hosts.  A load balancer can be used to distribute network and application traffic to individual containers by directly adding them or "linked" to a basic service.  A basic service that is "linked" will have all its underlying containers automatically registered as load balancer targets by Rancher.
+Rancher implements a managed load balancer using HAProxy that can be manually scaled to multiple hosts.  A load balancer can be used to distribute network and application traffic to individual containers by adding rules to target services. Any target service will have all its underlying containers automatically registered as load balancer targets by Rancher. With Rancher, it's easy to add a load balancer to your stack. We'll review the options for our load balancer as if you were using the UI and show examples using the [UI]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/adding-load-balancers/#adding-a-load-balancer-in-the-ui) and [Rancher Compose]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/adding-load-balancers/#adding-a-load-balancer-with-rancher-compose).
```

Comment:
Should we mention here that we are planning to expand the list of the LB providers? So it is clear that limiting it to haproxy is temporary.